src/bikesList.tsx

```diff
@@ -0,0 +1,222 @@
+import AddIcon from '@material-ui/icons/Add';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import Input from '@material-ui/core/Input';
+import {  stations} from "./stationList";
+import {BikeState,Bike, postBike} from "./Api/bikeApi";
+  const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+      ListSyle: {
+        overflowY: 'scroll',         
+      },
+      ListFont:{
+        color: 'white'
+      },
+      listSection: {
+        backgroundColor: 'inherit',
+      },
+      ul: {
+        backgroundColor: 'inherit',
+        padding: 0, 
+      },
+      deleteButton: {
+        backgroundColor: '#950740',
+        variant: 'contained'
+      }, 
+      container: {
+        display: 'flex',
+        flexWrap: 'wrap',
+      },
+      formControl: {
+        margin: theme.spacing(1),
+        minWidth: 120,
+      },  
+    }),
+  );
+  const themeWarning = createMuiTheme({
+    palette: {
+      primary:{
+        main:'#950740'
+      }
+    },
+  });
+  const themeListItem = createMuiTheme({
+    palette: {
+      primary:{
+        main:'#4E4E50'
+      }
+    },
+  });
+  let bicycles: Bike[] = [];
+  // bicycles.push({ID: 10,State: BikeState.Blocked,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 12,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  
+
+  
+const  BikeListPage = () => {
+    const classes = useStyles();
+    const [openSlidingWindow, setOpenSlidingWindow] = useState<boolean>(false);
+    const [open, setOpen] = useState<boolean>(false);
+    const [newBikeState, setState] = React.useState<number>(0);
+    const [newBikeStationID, setStation] = React.useState<number>(0);
+    const [list, setList] = React.useState<Bike[]>(bicycles);
+    const [selectedIndex, setSelectedIndex] = React.useState(-1);
+    const handleListItemClick = (
+      index: number,
+    ) => {
+      setSelectedIndex(index);
+    };
+  const handleChange = (event: React.ChangeEvent<{ value: unknown }>) => {
```

Comment:
nie wiadomo czego zmianę to handluje, zmień nazwę na `handleBikeStateChange` (jeśli dobrze rozumiem że to to robi)